<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoGlowne</class>
 <widget class="QMainWindow" name="OknoGlowne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zarządzanie projektami</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="nazwa_uzytkownika">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listaProjektow">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listaUczestnikow">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>221</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nazwa_projektu">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="uczestnik_projektu">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="terminarz">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>160</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoScrollMargin">
     <number>0</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AllEditTriggers</set>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>Data od</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data do</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="terminarz_opis">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>221</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="informacje_Qt"/>
   </widget>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="zmien_uzytkownika"/>
    <addaction name="separator"/>
    <addaction name="wyloguj"/>
   </widget>
   <widget class="QMenu" name="menuUzytkownik">
    <property name="title">
     <string>Użytkownik</string>
    </property>
    <addaction name="usun"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuUzytkownik"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="informacje_Qt">
   <property name="text">
    <string>Informacje o Qt</string>
   </property>
  </action>
  <action name="zmien_uzytkownika">
   <property name="text">
    <string>Zmień użytkownika</string>
   </property>
  </action>
  <action name="wyloguj">
   <property name="text">
    <string>Wyloguj</string>
   </property>
  </action>
  <action name="usun">
   <property name="text">
    <string>Usuń</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
